sql/recipe_db.sql

```diff
@@ -1,31 +1,178 @@
+
+/*táblárautalónév_oszlop elnevezéssel átláthatóbb*/
+
 CREATE DATABASE recipe_db;
 
 CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
 
+CREATE TABLE users (
+	user_id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+	user_username VARCHAR(100) NOT NULL UNIQUE,
+	user_email VARCHAR(100) NOT NULL UNIQUE,
+	user_password_hash VARCHAR(255) NOT NULL,
+	user_registration_date DATE DEFAULT CURRENT_DATE
+);
+
 CREATE TABLE recipes (
-	id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
-	name VARCHAR(100) NOT NULL,
-	description VARCHAR(255) NOT NULL,
-	vegetarian BOOLEAN DEFAULT false,
-	vegan BOOLEAN DEFAULT false,
-	glutenfree BOOLEAN DEFAULT false,
-	lactosefree BOOLEAN DEFAULT false,
-	sugarfree BOOLEAN DEFAULT false
+    recipe_id SERIAL PRIMARY KEY,
+    recipe_name VARCHAR(100) NOT NULL,
+    recipe_description VARCHAR(255) NOT NULL,
+    recipe_main_category_id SMALLINT,
+    recipe_img VARCHAR(255),
+    recipe_time_minutes SMALLINT,
```

Comment:
ennek stringnek kéne lennie, mert ilyenek lesznek benne, mint: 40 MIN

Reply:
recipe_main_category_id SMALLINT = 
 1 = easy
 2 = medium
 3 = hard
 Lehet select lista, radio button, checkbox vagy numeric input.
 
 recipe_time_minutes SMALLINT = Az időt egy numeric inputból be tudja küldeni, nem kell, hogy szöveget írjon mellé. Ha azt írja be, hogy 60 akkor értelem szerűen lekéred, hogy 60 és a szöveg mellette mindig perc (vagy nagyon túl bonyolítva át lehet váltani órákra, de máshol sem szokták.)
 
 recipe_serve_count SMALLINT = Serve count is sima numeric input, mert ott is csak azt választja ki, hogy a receptet hány főre készítette el.